frontend/src/pages/AllChatsPage.jsx

```diff
@@ -45,7 +79,11 @@ export default function AllChatsPage() {
     return (
         <div className="AllChatsPage FullHeight LightBackground">
             <ChatInboxHeader />
-            <ChatInbox chats={matches} users={matchedUsers} />
+            <ChatInbox chats={unblockedMatches} users={matchedUsers} />
+
+            {/* TODO: Make this a collapsable element & hide it if no blockedMatches are present */}
```

Comment:
Looks good! And yes, the collapsible element should be added in a future PR, that way the PR is focused just on that goal and integrating it here.